app/processing/delta/calculate-line-deltas.js

```diff
@@ -16,10 +16,12 @@ const calculateLineDeltas = (invoiceLines, defaultAgreementNumber) => {
       description: y.description,
       value: 0
     })
+    item.convergence = item.convergence || y.convergence
+    item.accountCode = item.accountCode || y.current ? y.accountCode : undefined
```

Comment:
Account code can be undefined at this point.  It's mapped at the end.

But for CS it comes in with 100% funded AP codes.  In order to calculate Delta correctly we need to know what the default 100% AP code is for the line.

Current means the line relates to the current payment request we're processing rather than a line from previous.

The idea here is we only add the account code to the group if it's from the current request.  If it's not or we've already completed that step then we just leave it.